src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,126 @@
 package core.basesyntax;
 
+import java.util.Map;
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private static final int CAPACITY_MULTIPLIER = 2;
+    private Node<K, V>[] table;
+    private int size;
+    private int capacity;
+    private int threshold;
+
+    @SuppressWarnings("unchecked")
+    public MyHashMap() {
+        table = (Node<K, V>[]) new Node[DEFAULT_CAPACITY];
+        capacity = DEFAULT_CAPACITY;
```

Comment:
look at the comment on line 93
and i`m not sure about "capacity = DEFAULT_CAPACITY". Is this right?